manifests/base/runner-manager/main/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: runner-manager
+spec:
+  template:
+    spec:
+      serviceAccountName: default
+      containers:
+        - args:
+            - 'runner_manager.main:app'
+            - '--host'
+            - 0.0.0.0
+            - '--port'
+            - '8000'
+          command:
+            - uvicorn
+          env:
+            - name: REDIS_OM_URL
+              valueFrom:
+                configMapKeyRef:
+                  key: REDIS_OM_URL
+                  name: redis-config
+          image: runner-manager
+          imagePullPolicy: IfNotPresent
+          name: runner-manager
+          ports:
+            - containerPort: 8000
+              name: http
+          resources:
+            limits:
+              cpu: 250m
+              memory: 1Gi
+            requests:
+              cpu: 250m
+              memory: 1Gi
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /_health
+              port: 8000
```

Comment:
```suggestion
              port: http
```

Reply:
why is it http

Reply:
Because you defined as http in the container specs [here](https://github.com/scality/runner-manager/pull/290/files/33de6879aca18d0ac038072ac579a0a60906b481#diff-92d2cf506a56a4f1702389dc6dc9c1dc94bb3fd3a87a1382775a5061dc4b8963R27)

Reply:
ok thanks